VideoLocker/build.gradle

```diff
@@ -311,6 +311,7 @@ android {
     lintOptions {
         abortOnError true
         showAll true
+        enable 'ContentDescription'
```

Comment:
Could we actually add this to the "error" scope? We want to ensure that these errors cause failures on Travis